setup.py

```diff
@@ -92,22 +87,9 @@ def build_extensions():
         config["extra_compile_args"].extend(("-fopenmp", "-shared"))
         config["extra_link_args"].extend(("-fopenmp",))
 
-    if EIGEN_INCLUDE_DIR:
-        config["include_dirs"].append(EIGEN_INCLUDE_DIR)
-    else:
-        config["include_dirs"].extend(
-            ("/usr/include/eigen3", "/usr/local/include/eigen3")
-        )
-
-    if USE_OPENBLAS:
-        config["extra_compile_args"].append("-lopenblas")
-
-    if USE_LAPACK:
-        config["extra_compile_args"].extend(("-DLAPACKE=1", "-llapack"))
-
     return cythonize(
         [Extension("libwalrus", **config)],
-        compile_time_env={"_OPENMP": USE_OPENMP, "LAPACKE": USE_LAPACK},
+        compile_time_env={"_OPENMP": USE_OPENMP},
```

Comment:
```suggestion
        compile_time_env={},
```
I suppose this can be removed as well?

btw, the `setup.py` is so much cleaner without eigen 🙌